go.mod

```diff
@@ -406,13 +412,16 @@ replace (
 	github.com/thecodeteam/goscaleio => github.com/thecodeteam/goscaleio v0.1.0
 	github.com/tmc/grpc-websocket-proxy => github.com/tmc/grpc-websocket-proxy v0.0.0-20170815181823-89b8d40f7ca8
 	github.com/ugorji/go/codec => github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8
+	github.com/urfave/cli => github.com/urfave/cli v1.20.0
 	github.com/urfave/negroni => github.com/urfave/negroni v1.0.0
 	github.com/vishvananda/netlink => github.com/vishvananda/netlink v0.0.0-20171020171820-b2de5d10e38e
 	github.com/vishvananda/netns => github.com/vishvananda/netns v0.0.0-20171111001504-be1fbeda1936
 	github.com/vmware/govmomi => github.com/vmware/govmomi v0.20.1
 	github.com/xiang90/probing => github.com/xiang90/probing v0.0.0-20160813154853-07dd2e8dfe18
 	github.com/xlab/handysort => github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1
 	github.com/xordataexchange/crypt => github.com/xordataexchange/crypt v0.0.3-0.20170626215501-b2862e3d0a77
+	go.etcd.io/bbolt => go.etcd.io/bbolt v1.3.3
+	go.etcd.io/etcd => go.etcd.io/etcd v0.0.0-20190917205325-a14579fbfb1a
```

Comment:
Should be the `v3.4.1` version here?

Reply:
@liggitt, based on your comments here, https://github.com/kubernetes/kubernetes/pull/81434#issuecomment-522394787, is this intended?

Reply:
unfortunately, yes. etcd (like the k8s.io libraries) is using a semver tag >1.x with a go.mod file, and is not using semantic import versioning (e.g. naming their module `go.etcd.io/etcd/v3`)

go considers that invalid, and that means go will not honor/preserve the semver tag and will resolve the version to a pseudo-version that reads as `v0.0.0-<commit-date>-<sha>`